Assets/MRTK/Providers/XRSDK/XRSDKDeviceManager.cs

```diff
@@ -62,14 +62,14 @@ public virtual bool CheckCapability(MixedRealityCapability capability)
         /// </summary>
         /// <param name="loaderName">The string name to compare against the active loader.</param>
         /// <returns>True if the active loader has the same name as the parameter.</returns>
-        protected virtual bool IsLoaderActive(string loaderName) => XRGeneralSettings.Instance.Manager.activeLoader != null && XRGeneralSettings.Instance.Manager.activeLoader.name == loaderName;
+        protected virtual bool IsLoaderActive(string loaderName) => XRGeneralSettings.Instance?.Manager?.activeLoader != null && XRGeneralSettings.Instance.Manager.activeLoader.name == loaderName;
```

Comment:
Both `XRGeneralSettings.Instance` and `.Manager` reference a ScriptableObject, so we should use an explicit null check instead of `?.` to ensure we're using Unity's overloaded null check for Unity objects

![image](https://user-images.githubusercontent.com/3580640/111533090-a2363a00-8723-11eb-9c58-b4decf727062.png)